Dockerfile

```diff
@@ -0,0 +1,111 @@
+#################################### Build front
```

Comment:
This is clearer but there was a reason the Dockerfile wasn't structure like this before: the image size. 

As far as I know, multiple RUN statement will make the image size much bigger than what it is now because of the layer approach to image building in docker. Considering the user's feedback I see a lot of people concerned about image size rather than readability of the Dockerfile.

Reply:
i will improve it, but the main point of the file was to use relative code instead of the one from your website.